idowuadeleke/BANKA
# Chore(created status code):changed from 200 to 201

### What does this PR do?
Implements created status code feedback 

### Description of Task to be completed?
- status should return 201 when a new user is created
  `POST localhost:3000/api/v1/auth/signup`

### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ch-change-created-status-code-to-201-165381141`

- run `npm install`

- run `npm run start`

- make a request to `POST localhost:3000/api/v1/auth/signup`
  NB see screenshot below on the request data to pass

### Any background context you want to provide?
NA
### What are the relevant pivotal tracker stories?
[#165381141](https://www.pivotaltracker.com/story/show/165381141)

### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40268693/56213020-258c1400-6053-11e9-87b5-1a2b2884f39b.png)
